### DISCUSSION TITLE
Not able to get debug logs(UART) with nRF5340-DK

### DISCUSSION THREAD

**Discussion by govindsi** [contiki-ng/contiki-ng#1707] [Q&A] [answered]
I am using nRF5340-DK board and unable to get the debug logs with existing sample application.

Steps:

make TARGET=nrf clean
make TARGET=nrf
make TARGET=nrf hello-world.upload-all

nrfjprog -f NRF53 --coprocessor CP_APPLICATION  --snr 960134989 --sectorerase --program build/nrf/nrf5340/dk/application/hello-world.hex
Parsing image file.
Applying system reset.
Verified OK.
nrfjprog -f NRF53 --coprocessor CP_APPLICATION  --snr 960134989 --reset
Applying system reset.
Run.
Done
make[1]: Leaving directory '/home/govind/work/NRF/contiki-ng/examples/hello-world'
rm hello-world.o build/nrf/nrf5340/dk/application/hello-world.i16hex

picocom -fh -b 38400 --imap lfcrlf /dev/ttyACM0 

Is there explicit config needs to be enabled for getting logs over UART?

NRF52840-DK is working fine using above steps(TARGET=nrf52dk).

**Comment by alexstanoev** [comment, marked as answer]
The nRF5340 DK enumerates as 3 serial ports - have you checked all of them (especially the last one)?
IIRC the baud rate should be 115200 by default.

**Comment by govindsi** [comment]
Thanks, it worked with the last enumerated com port.
picocom -fh -b 115200 --imap lfcrlf /dev/ttyACM2

**Reply by sebastianrameshs** [reply]
I face problem in k_sem_take()   i build a nrf3540 board its successfully running in peripheral uart and some sample programs. while debugging peripheral connectionless rx and tx its stop executing in k_sem_take(). its show no complication error but its not flashings in nrf5340. i need to use AOA &AOD function and calculate the direction finder. will u help me out i stuck for a week .

**Reply by sebastianrameshs** [reply]
how to see the print in this nRF5340 dk in vscode sample code also not execting
